docs/UserGuide.md

```diff
@@ -140,40 +147,50 @@ Curious about the various commands? Refer to the [Features](#features) below to
 
 ### Familiarising yourself with the interface
 
-To familiarise yourself with our user interface, please refer to the following diagram and table to get comfortable with the application.
+To familiarise yourself with our user interface, please refer to the following diagram and table to get comfortable with
+the application.
 
 ![UserInterfaceExplanation](images/ug/UserInterfaceExplanation.png)
 
 |      `Component`       |                                                       `Explanation`                                                        |
 |:----------------------:|:--------------------------------------------------------------------------------------------------------------------------:|
-|    **List Display**    |            where you view your entry lists (Expenditure or Income) <br> - can be modified by the `view` command            |
-|   **Graph Display**    | where you view your entries in a graphical overview (Pie Chart or Line Graph) <br> - can be modified by the `view` command |
-| **Command Input Box**  |                                       where you enter your commands to use PennyWise                                       |
-| **Command Output Box** |                                  where you see the output of your commands from PennyWise                                  |
+|    **List
+Display**    |            where you view your entry lists (Expenditure or Income) <br> - can be modified by the `view` command            |
+|   **Graph
+Display**    | where you view your entries in a graphical overview (Pie Chart or Line Graph) <br> - can be modified by the `view` command |
+| **Command Input
+Box**  |                                       where you enter your commands to use PennyWise                                       |
+| **Command Output
+Box** |                                  where you see the output of your commands from PennyWise                                  |
```

Comment:
Not sure why the formatting for this is adjusted, looks a bit weird when viewed in markdown

![image](https://user-images.githubusercontent.com/63844743/199515282-97ea37b2-7edd-4240-a5a2-860a65ea3e2d.png)